src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -475,8 +475,21 @@ - (void)subscribeWithQueue:(dispatch_queue_t)queue
                                   }];
 }
 
+static NSDictionary<NSString *, id> * _MakeDataValueDictionary(NSString * type, id _Nullable value, NSNumber * _Nullable dataVersion)
```

Comment:
Opening up a comment so the PR holds until @jtung-apple has resolved this comment.

Reply:
Resolved in the latest commit.